### ISSUE TITLE
Fern not detecting ISO 8601 conform timestamp as ISO conform example

### ISSUE THREAD

**Issue by fisehara** [fern-api/fern#1447]
```
Expected example to be an ISO 8601 timestamp. Example is: "2023-02-16T16:47:38.259Z"
```

Example date string is generated in typescript with: `return new Date().toISOString()`

minimal yaml example:
```
types:
  Application:
    properties:
      created_at:
        type: optional<datetime>
      modified_at:
        type: optional<datetime>
      id:
        type: long
        docs: The unique identifier for a Application
      name:
        type: string
      note:
        type: string
 service:
  auth: false
  base-path: /example
  endpoints:
    ReadAllApplication:
      path: /Application
      method: GET
      response: map<string,list<Application>>
      request:
        name: ReadallApplication
        query-parameters:
          $filter: optional<string>
          $select: optional<list<string>>
          $expand: optional<string>
          $top: optional<integer>
          $count: optional<integer>
      examples:
        - query-parameters:
            $select:
              - created_at
              - modified_at
              - id
              - name
              - note
          response:
            body:
              d:
                - created_at: 2023-02-16T16:47:38.259Z
                  modified_at: 2022-07-05T06:15:10.690Z
                  id: 60390
                  name: odemzoajofccqprljwgo
                  note: ibzgphjfjozqwrzrtroi
                - created_at: 2023-02-16T16:47:38.259Z
                  modified_at: 2022-07-05T06:15:10.690Z
                  id: 60390
                  name: odemzoajofccqprljwgo
                  note: ibzgphjfjozqwrzrtroi
                  
```

**Comment by dsinghvi** [comment]
@fisehara thanks for reporting this bug, we'll take a look at fixing it.

**Comment by zachkirsch** [comment]
@fisehara this is fixed in `0.6.6`. You can run `fern upgrade` to grab the latest version

**Comment by fisehara** [comment]
@zachkirsch I've tested it and it is fixed, thanks a lot!
